### DISCUSSION TITLE
"Apply" function at Settings

### DISCUSSION THREAD

**Discussion by Emilot** [jcorporation/myMPD#446] [Q&A] [answered]
JC can you sort explain what "Apply" means in Settings modal?

I understand it in "Themes" at Appearance card, when someone want to "take a look" at what the change will do and hit "Apply" to see and to Save it or not. By the way if someone hit "Apply" and do not want to Save it, when he/she hit Cancel nothing happens...it continues to use whatever change he/she applied earlier (bug).

But what is the meaning in the rest cards?

Thank you.

**Comment by jcorporation** [comment, marked as answer]
Apply does all what Save does, but not closes the settings dialog.

**Reply by Emilot** [reply]
Hmmm...So why someone use it?

What is the practical meaning of it? Sorry to ask but i want to understand if there is something misunderstood about it and mainly because i want to explain it to others who ask me.

**Reply by jcorporation** [reply]
What you describes in your initial questions was a "Theme preview" that does not exist. I personally use the "Apply" Button, if I want to apply a setting without closing the dialog, e.g. enable a tag -> apply -> select tag as browse tag, or to try different theme colors.

**Reply by Emilot** [reply]
For colors is understood. But when you choose tag and hit Apply, how do you see the results of this action? You must close the Settings and see the results in the corresponding card.

As i understand it, Apply is useful only in Appearance card.

**Reply by jcorporation** [reply]
If you select a tag in the "Tags to use" select you must press apply before you can select this tag in the "tags to search" or "tags to browse" select.

But yes, the apply button is not useful in other cased

**Reply by Emilot** [reply]
Thank you JC for the explanations.

👍
